src/abi/ethereum.rs

```diff
@@ -1,4 +1,5 @@
 // ethereum style ABIs
+use crate::ast::StructType;
 use crate::sema::ast::{Namespace, Parameter, Type};
```

Comment:
```rust
use crate::sema::ast::{Namespace, Parameter, StructType, Type};
```

We should probably remove the `use sema::ast;` in `src/lib.rs` so the ast doesn't have two paths

Reply:
I removed the `use sema::ast;` from `src/lib.rs`, but it entailed many other changes on a bunch of files.